src/Accounts/Accounts/help/Get-AzAccessToken.md

```diff
@@ -8,7 +8,8 @@ schema: 2.0.0
 # Get-AzAccessToken
 
 ## SYNOPSIS
-Get raw access token. When using -ResourceUrl, please make sure the value does match current Azure environment. You may refer to the value of `(Get-AzContext).Environment`.
+Get secure raw access token. When using -ResourceUrl, please make sure the value does match current Azure environment. You may refer to the value of `(Get-AzContext).Environment`.
+> **_NOTE:_**  The current default output type `PSAccessToken` is going to be deprecate. Please always use `-AsSecureString` to avoid the upcoming breaking change. After that the output turns to be `PSSecureAccessToken`, which contains `token` as a `SecureString`. Please refer to [Unprotect-SecureString](https://github.com/Azure/azure-powershell/blob/main/src/Aks/Aks.Autorest/utils/Unprotect-SecureString.ps1) for how to convert from `SecureString` to plain text.
```

Comment:
It's unlikely that we will maintain this link in the future. My suggestion is contribute to our official FAQ https://learn.microsoft.com/en-us/powershell/azure/faq?view=azps-12.4.0

Reply:
Probably needs some rewriting. The change of the Token type is more important than the output type. I suggest not mentioning the output type.

Reply:
Done